packages/opencensus-web-scripts/README.md

```diff
@@ -45,6 +46,28 @@ these two bundles to do instrumentation first and then an export more lazily:
     downloaded and parsed doesn't need to compete with resources for the main
     page load.
 
+If your application already depends on `Zone.js`, the case for `Angular`, use:
+
+* **`./dist/tracing-all-without-zone.js`** - This script combines all the functionality of
```

Comment:
What about calling this `tracing-all-zone-peer-dep`? The phrase "without-zone" makes me think it might not be dependent on Zone.js at all, but in fact it is, just indirectly